src/style.css

```diff
@@ -0,0 +1,98 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+}
+
+.searchInput {
+  color: #3d4e61;
+  width: 100%;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  background: transparent;
+  /* background-color: lightblue; */
+}
+
+.searchInputBig {
+  font-weight: 300;
+  font-size: 16px;
+  height: 70px;
+  padding: 0 0 0 60px;
+}
+
+.searchInputSmall {
+  font-weight: 400;
+  font-size: 14px;
+  height: 42px;
+  padding: 0 0 0 33px;
+}
+
+.searchBar {
+  width: 100%;
+  box-sizing: border-box;
+  margin: 20px 0 0 ;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background: url(images/Search.svg) no-repeat;
+
+  /* background-color: greenyellow; */
+
+}
+
+.searchBarBig {
+  height: 70px;
+  background-position-x: 26px;
+  background-position-y: 24px;
+  background-size: 19px;
+}
+
+.searchBarSmall {
+  height: 42px;
+  background-position-x: 13px;
+  background-position-y: 15px;
+  background-size: 11px;
+}
+
+.searchInput::placeholder {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  color: #3d4e61;
+  width: 100%;
+  height: 60px;
+  vertical-align: middle;
+ 
+
+  /* background-color: pink; */
+}
+
+.searchInputSmall::placeholder {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+  color: #3d4e61;
+  font-style: normal;
+  vertical-align: middle;
+
+}
+
+.searchBar:hover {
+  border: 1px solid #e1e7ed;
```

Comment:
No need to give border here as it is same as default

Reply:
resolved. thanks.